cask.py

```diff
@@ -1683,5 +1683,19 @@ def __init__(self, *args, **kwargs):
 
 class Points(Object):
     """Points I/O Object subclass."""
+    _sample_class = alembic.AbcGeom.OPointsSchemaSample
     def __init__(self, *args, **kwargs):
         super(Points, self).__init__(*args, **kwargs)
+
+    def set_positions(self, *args):        
```

Comment:
Please see Xform.set_scale, https://github.com/alembic/cask/blob/master/cask.py#L1617

This should probably accept both a V3fArray or a list of values, and check the type.

```suggestion
    def set_positions(self, positions):        
```